src/main/scala/no/ndla/imageapi/controller/ImageController.scala

```diff
@@ -145,8 +146,9 @@ trait ImageController {
 
     get("/:image_id", operation(getByImageId)) {
       val imageId = long("image_id")
-      imageRepository.withId(imageId) match {
-        case Some(image) => converterService.asApiImageMetaInformationWithApplicationUrl(image)
+      val language = params.get("language").getOrElse(AllLanguages)
+      imageRepository.withId(imageId).flatMap(image => converterService.asApiImageMetaInformationWithSingleLanguage(image, language)) match {
```

Comment:
Dette er en "breaking change" som vil påvirke en hel del andre ting, derfor er det best om dette inngår i en V2-kontroller (tilsvarende slik vi snakket om for article-api). Husk også å oppdatere swagger dokumentasjonen med riktig modell som returneres fra et endepunkt:)